presto-main/src/main/java/com/facebook/presto/resourcemanager/ResourceManagerConfig.java

```diff
@@ -278,4 +279,16 @@ public ResourceManagerConfig setRunningTaskCountFetchInterval(Duration runningTa
         this.runningTaskCountFetchInterval = runningTaskCountFetchInterval;
         return this;
     }
+
+    public boolean getHeartbeatHttpEndpointEnabled()
```

Comment:
nit : `isHeartbeatHttpEndpointEnabled()`
You can simplify and just say http enabled as well.